Makefile

```diff
@@ -21,3 +20,7 @@ clean:
 .PHONY: clean
 
 GO_TEST_PACKAGES :=./pkg/... ./cmd/...
+
+verify-deps:
```

Comment:
That's wrong, you should disable this in https://github.com/openshift/release/blob/1004081a037d723a30341af9ac1e5db584496b14/ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml#L41-L44 compare https://github.com/openshift/release/pull/5313. These need to be re-written so it's not worth faking this.

Reply:
Fixed in latest push.

https://github.com/openshift/release/pull/5573 removes `verify-deps` from ocm-o.

Reply:
https://github.com/openshift/release/pull/5574 removes `verify-deps` from ocm